Core1Platform/AdministratingCore1/00_AdministratingCore1.md

```diff
@@ -0,0 +1,39 @@
+# Administrating the Core1 Platform
+<!--- Links getestet-->
+In this part of the *Actindo Core1 Platform* documentation, you will find the following chapters:
+
+- [Main account and sandboxes](./01_MainAccountSandbox.md)   
+
```

Comment:
H2 hinzufügen?

- Main account
- Sandbox
- Differences between sandbox and main account (oder Comparison between sandbox and main account)

Reply:
Es gibt kein H2 in diesem Text